projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -141,7 +141,7 @@ int main()
 			ad77681_spi_read_adc_data(adc_dev, adc_data,
 						  AD77681_CONTINUOUS_DATA_READ);
 			printf("[ADC DATA]: 0x");
-			for(i = 0; i < sizeof(adc_data) / sizeof(uint8_t); i++) {
+			for(i = 0; i < sizeof(adc_data); i++) {
```

Comment:
shouldn't this be ARRAY_SIZE ?

Reply:
ok i checked the code, it doesn't really matter here, fine by me

Reply:
Looking at it now this change seems unnecessary. It doesn't add or change anything meaningful. I would propose to either change it to ARRAY_SIZE, like most other for loop limits, or remove the change.